CONTRIBUTING.md

```diff
@@ -179,6 +179,23 @@ TEST_SET=jvm-integration  \
 
   This will bootstrap pants and resolve all the dependencies or else you will get an `ExecutionException` exception for exceeding 30s timeout.
 
+## Typical Major Release Upgrade Procedures
+
+With the [2019.2 upgrade](https://github.com/pantsbuild/intellij-pants-plugin/commit/293f24969312f23219739505273bb177fca29283) as a reference:
+1. Bump the version range in `resources/META-INF/plugin.xml`, especially if the newer version is backward incompatible.
+2. Bump the build number and version in `scripts/prepare-ci-environment.sh`, so tests can be run against the newer version.
+3. Update `testData/external-system-test-api.jar` (Not required if there's no breaking change). After this point, there should be no more compile errors.
+
+    1. Checkout https://github.com/JetBrains/intellij-community
+    2. Checkout the tag that corresponds to the new version. E.g. `idea/192.5728.98`
+    3. Open the project in IntelliJ (takes some minutes for IntelliJ to auto import the project)
+    4. Configure an `external-system-test-api` artifact like this
+    ![](images/external-system-test-api_configure.png)
```

Comment:
It would help to have more details on how to configure it "like this."

The + button in Artifacts gives me a few options that might be relevant: JAR > Empty, JAR > From modules with dependencies..., and Other (also some Android and JavaFx ones that are obviously inapplicable).

I tried JAR > Empty. When I hit the + button within the configuration and select Module Test Output (based on the icon in the screenshot), the only module option I'm given is `intellij-pants-plugin`. I admit I haven't yet followed all the instructions above... is that why it's not giving me the option for `intellij.platform.externalSystem.impl`?

Reply:
Yeah I can make it more step by step.

Reply:
Made it a copy paste of xml instead, since it's likely easier.